tests/data/qa/multiqc_data/rna/multiqc_sources.txt

```diff
@@ -0,0 +1,3 @@
+Module	Section	Sample Name	Source
+Picard	RnaSeqMetrics	SA622799	/Users/gfeng/Documents/Work/ICGC-ARGO-WORKFLOWS/rnaaln/update_prep_metrics/nf-core-rnaaln/work/db/878450a93307149b3ab3b8e29cae66/5/TCRB-CA.DO262498.SA622799.csort.markdup.rna_metrics
```

Comment:
Remove/generalize user directory

Reply:
The directory path does not affect testing. In the danaln testing data It is similar style. Maybe we can leave it as it is?